charger/phoenix-ev-eth.go

```diff
@@ -177,7 +155,7 @@ func (wb *PhoenixEVEth) currentPower() (float64, error) {
 		return 0, err
 	}
 
-	return wb.decodeReading(wb.powerScale, b), err
+	return wb.decodeReading(b, 1e4), err
```

Comment:
Ich verstehe das nicht. Laut Doku ist die Ausgabe Integer in Watt. Warum sollte hier ein Scaler sein? Auf welche Doku beziehst Du Dich?

Reply:
Ich beziehe mich hier ganz einfach auf den tatsächlichen Registerinhalt. Pures Testen am echten Objekt.
Die Doku hat mir zu viele Fehler.
Das ist eher so ein grober Anhaltspunkt.